### ISSUE TITLE
.COM works but .EXE hangs?

### ISSUE THREAD

**Issue by Zweitaktmotor** [joncampbell123/dosbox-x#3099]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

Hi, 

Just found DOSbox-X and installed it. Seems a great idea since I have lots of DOS programs from 30 years ago. Ultimately, I am hoping to get Geoworks/Breadbox Ensemble working again. :-)

But I need to get a bit of help to get going, please:

I seem to be able to run .COM files, but if I call up an .EXE file, my cursor just jumps to the next line and nothing happens.

In this screenshot:

![Screenshot from 2021-11-22 20-57-26](https://user-images.githubusercontent.com/7682684/142934369-8499c939-1b33-435f-829a-08beafe981ef.png)

I am trying to run a simple clock utility that used to be my standard PC test as it seemed to never fail unless something was seriously wrong.

I am sure this is just a setting that needs to be adjusted, but which one?

Many thanks.

**Comment by Wengier** [comment]
You did not post the EXE file(s) you used, but I cannot reproduce this problem with the EXE files I have, including the built-in ones. Can you try for example the EXE programs included on the Z drive, such as those EXE programs included in the Z:\DOS and Z:\BIN directories?

**Comment by Zweitaktmotor** [comment]
Not sure yet, but I think I found the problem. The tools on your Z: drive do work indeed and I found a few programs of my own that also work, but others that don't.

I managed to find the original ZIP file for one of them, re-installed it and now it does work. This reminds me that I used to do some geeky stuff in the early 1990s.

My first PC was an 8086 laptop with 1 MB RAM, a 720kb floppy drive and a 20 MB hard disk. :-)))

Space was precious and I always tried to make everything as small as possible: Find the smallest drivers, reformat all my floppies to a non-standard 800kb - and I seem to remember that I had a procedure that could shave off a few bytes from executables.

This could be the problem. I will investigate further and let you know if I find a program that does not run in its original form. First I have to track down the originals as I cannot remember the procedure and which files I applied it to. :-)

Many thanks.

On a side note: Does DOSbox-X run on ARM architecture? I have a Linux Pinephone. It's all still experimental, but several OSs are working on it (sort-of). They just developed a fold-out keyboad. It would be cool to have a "DOS PC" in my pocket. The old Atari Portfolio came close, but was too underpowered even for my needs at the time.

Kind regards.
